gpt_index/io.py

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
this file is kinda confusing. we have a `constants.py` file, why not store there?

Reply:
Agreed.